lib/membrane_rtc_engine/engine.ex

```diff
@@ -312,6 +312,14 @@ defmodule Membrane.RTC.Engine do
     Pipeline.call(rtc_engine, :get_endpoints)
   end
 
+  @doc """
+  Returns number of forwarded tracks in RTC Engine.
+  """
+  @spec get_forwarded_tracks(rtc_engine :: pid()) :: integer()
```

Comment:
[nitpick]
```suggestion
  @spec get_forwarded_tracks_number(rtc_engine :: pid()) :: integer()
```

alternatively:
get_forwarded_track_count
get_num_forwarded_tracks